app/src/main/res/layout/activity_help_page.xml

```diff
@@ -101,30 +101,30 @@
                 android:layout_marginBottom="20dp">
 
 
-            <ImageView
+                <ImageView
+                    android:id="@+id/reject"
                     android:layout_width="70dp"
                     android:layout_height="70dp"
-                    android:layout_marginHorizontal="70dp"
                     android:layout_gravity="start"
+                    android:layout_marginHorizontal="70dp"
+                    android:onClick="refuseHelp"
                     android:src="@drawable/cancel_button"
-                    android:id="@+id/reject"
-                    android:onClick="goToMainPage"
-                    app:layout_constraintTop_toTopOf="parent"
                     app:layout_constraintBottom_toBottomOf="parent"
-                    app:layout_constraintStart_toStartOf="parent" />
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="parent" />
 
-            <ImageView
+                <ImageView
+                    android:id="@+id/accept"
```

Comment:
This needs to be removed when you merge because when the user accepts to help I have changed the listener to redirect the user to the chat activity.